xbmc/Application.cpp

```diff
@@ -2482,8 +2482,14 @@ void CApplication::FrameMove(bool processEvents, bool processGUI)
   {
     m_skipGuiRender = false;
     int fps = 0;
+
+    #if defined(TARGET_RASPBERRY_PI) || defined(HAS_IMXVPU)
```

Comment:
please, please, please move those to the very left. what is the benefit of mixing code with pre-processor statements?